stock_location_restrict_group/models/stock_location.py

```diff
@@ -26,7 +26,6 @@ def _compute_restricted_group(self):
         for location in self.filtered(lambda l: l.group_restricted):
             quants = self.env['stock.quant'].search(
                 [('location_id', '=', location.id)])
-            location.restricted_group = False
             rest_group = quants.mapped('history_ids').filtered(
```

Comment:
I've added search on location but let 'restricted_group' not stored because of trigger complexity.